var/spack/repos/builtin/packages/py-quast/package.py

```diff
@@ -27,6 +31,8 @@ class PyQuast(PythonPackage):
     depends_on("python@2.5:,3.3:")
```

Comment:
Perl and Python are currently build/link deps, meaning they aren't available at run-time. Is this what you want?

Reply:
What do you mean by that?

Reply:
See https://spack.readthedocs.io/en/latest/packaging_guide.html#dependency-types